pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1379,6 +1388,19 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     return path;
   }
 
+  protected async sudoRequiresPassword() {
+    try {
+      await childProcess.spawnFile('sudo', ['-n', 'true'], { stdio: ['ignore', 'pipe', 'pipe'] });
```

Comment:
Please use the long form (`sudo --non-interactive`) in scripts etc.  And for clarity, `/usr/bin/true` over `true`.

Reply:
I was convinced that `sudo` on Linux didn't support `--non-interactive`, but it seems to be supported since sudo 1.8.8, which was released in 2013, so maybe that is good enough for us.